include/zpc/aes_key.h

```diff
@@ -29,10 +29,19 @@ extern "C" {
 # define ZPC_AES_KEY_TYPE_CCA_DATA     1
 # define ZPC_AES_KEY_TYPE_CCA_CIPHER   2
 # define ZPC_AES_KEY_TYPE_EP11         3
+# define ZPC_AES_KEY_TYPE_PVSECRET     9 
 
 # define ZPC_AES_KEY_REENCIPHER_OLD_TO_CURRENT    1
 # define ZPC_AES_KEY_REENCIPHER_CURRENT_TO_NEW    2
 
+typedef enum {
+	ZPC_AES_SECRET_TYPE_NOT_SET = -2,
+	ZPC_AES_SECRET_TYPE_INVALID = -1,
+	ZPC_AES_SECRET_AES_128 = 0x04,
```

Comment:
Is there a reason why this starts a 0x04, and not at 0x01 ?

Reply:
Ah OK, maybe a comment would help to explain that these are the architected secret types.
Same for the EC key types.

Reply:
Similar hex codes show up in ecc_key.c for the EC type secrets. The hex codes always directly relate to the secret types.

Reply:
Added comment.